moped-etl/prefect/moped_test_create_flow.py

```diff
@@ -229,28 +230,36 @@ def remove_activity_log_lambda():
     )
     undeploy_api = remove_api_task(command=decommission_api_command)
 
+with Flow(
+    "Moped Netlify Commission", run_config=UniversalRun(labels=["moped", hostname])
+) as netlify_commission:
+
+    basename = Parameter("basename")
+
+    build = trigger_netlify_build(branch=basename)
+    is_ready = netlify_check_build(branch=basename, build_token=build)
 
 if __name__ == "__main__":
     print("main()")
 
-    basename = "flh-parameter-test"
+    basename = "netlify-test-deployment"
```

Comment:
`basename` is a branch name here; I really owe us the slugging system and to clearly define how that's going to work. The more we develop without it, the more it will end up being made to suit our needs - but we're close to needing it PDQ.